app/jobs/send_reject_resolution_reminder_notification_job.rb

```diff
@@ -1,37 +0,0 @@
-class SendRejectResolutionReminderNotificationJob < ApplicationJob
```

Comment:
oh nice i did some consolidation in this ticket as well:
https://github.com/codeforamerica/vita-min/pull/5853/files#diff-6c8f13777b0e7566d705569e8a9f12dbe414e969c87bd95be6fef31857a3fcb8